### PR TITLE
Maintenance announcement bar - Jan '24

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

This pull request adds maint window announcement bar

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [x] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -234,6 +234,10 @@ module.exports = {
       },
     // SEO Global Metadata
     metadata: [{name: 'keywords', content: 'sumo logic, documentation, tutorials, quickstarts'}],
+    announcementBar: {
+      id: 'announcementBar',
+      content: `Sumo Logic will be down for maintenance beginning Jan 10, 2024 12 a.m. PST. Follow our <a href="https://status.sumologic.com">status page</a> for updates.`,
```
Comment (kimsauce) [inline docusaurus.config.js:239]:
```suggestion
      content: `Our systems will undergo maintenance on Jan 10, beginning at 12 a.m. PST, and ending Jan 11. Access to certain portions of Sumo Logic may be temporarily unavailable. Follow our <a href="https://status.sumologic.com">status page</a> for updates.`,
```

File: docusaurus.config.js [hunk 2]
```diff
@@ -234,6 +234,10 @@ module.exports = {
       },
     // SEO Global Metadata
     metadata: [{name: 'keywords', content: 'sumo logic, documentation, tutorials, quickstarts'}],
+    announcementBar: {
+      id: 'announcementBar',
+      content: `Our systems will undergo maintenance on Jan 10, beginning at 12 a.m. PST, and ending Jan 11. Access to certain portions of Sumo Logic may be temporarily unavailable. Follow our <a href="https://status.sumologic.com">status page</a> for updates.`,
```
Comment (kimsauce) [inline docusaurus.config.js:239]:
```suggestion
      content: `Our systems will undergo maintenance on Jan 10, beginning at 12 a.m. PST, and ending Jan 11. During this time, you may experience minor service disruption. Follow our <a href="https://status.sumologic.com">status page</a> for updates.`,
```

**Review by jpipkin1 — APPROVED** [review]
Looks good, @kimsauce.

My only thought is that the announcement doesn't stand out. What do you think about making the background color yellow like you did for the re:Invent banner (PR [#3326](https://github.com/SumoLogic/sumologic-documentation/pull/3326/files))?

**Comment by jpipkin1** [comment]
The color on the banner looks good, @kimsauce.

**Comment by kimsauce** [comment]
> Looks good, @kimsauce.
> 
> My only thought is that the announcement doesn't stand out. What do you think about making the background color yellow like you did for the re:Invent banner (PR [#3326](https://github.com/SumoLogic/sumologic-documentation/pull/3326/files))?

thank you! great catch.

**Pull request merged**